Globals.c

```diff
@@ -113,4 +113,10 @@ DWORD RandOffset(DWORD dwOrigValue, int nMilliseconds)
 		dwNewValue += (((rand() % g_UserSettings.Offset)) * nMultiplier); // Variance by 5 units
 	}
 	return dwNewValue;
+}
+
+// Generate a random single digit using different method to obfuscate static values like IDLEINTERVAL
+
+int ranum() {

```

Comment:
should return uint8_t for strict smallest type defined value, consider inline declaration

Reply:
Bot.h:
`uint8_t RandGen();`

Bot.c:
`__inline uint8_t RandGen()
{
        /* ... */
        return (uint8_t)/* ... */;
}`